src/app/auth/components/AuthModalComponent.js

```diff
@@ -46,6 +45,13 @@ class AuthModalComponent extends Component {
 
     if (!hasMetamask) {
       body = <Button type="link" onClick={openWallets}>{strings.get_metamask}</Button>;
+    } else if (network === 'invalid') {
+      body = (
+        <div>
+          <p><strong>{strings.network_mismatch}</strong></p>
```

Comment:
Isn’t it better to display the same text we display in the new indicator?

Reply:
Yes, I can change this.

That string is in the `indicator-light` branch which is pending the translations. I will wait for that branch to be merged, then update line 51 to use the invalid string and push the commit.

Reply:
Resolved with commit [d19b6d6](https://github.com/rnsdomains/rns-manager-react/pull/124/commits/d19b6d67ce40a18ad3f9848a0cbec75bc19ca1b5)